tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -570,8 +587,8 @@ def log_phrase(x,y):
         return a
 
     f2 = epyccel(log_phrase, language = language)
-    x = uniform(low=min_float, high=1e6)
-    y = uniform(low=min_float, high=1e6)
+    x = uniform(low=abs(min_float), high=1e6)
+    y = uniform(low=abs(min_float), high=1e6)
```

Comment:
abs(min_float) is not a good value for this:
```
In [6]: abs(finfo('float').min) == finfo('float').max
Out[6]: True
```

Reply:
Wow :sweat_smile:

Reply:
ah yes.

Reply:
OK then, we have to use `finfo(float).tiny`

Reply:
@mostafaMamoni Sorry for the confusion.